docs/design/external_traffic_policy.md

```diff
@@ -0,0 +1,128 @@
+type services. 
+
+## Ingress Traffic 
+
+This section will cover the networking entities hit when traffic ingresses a cluster via a service to either host
+networked pods or cluster networked pods
+
+### External Source -> Service -> OVN pod
+
+This case is the same as normal shared gateway traffic ingress, meaning the externally sourced traffic is routed into 
+OVN via flows on breth0, except in this case the new local load balancer is hit on the GR, which ensures the ip of the 
+client is preserved  by the time it gets to the destination Pod. 
+
+```text
+          host (ovn-worker, 172.18.0.3) 
+           |
+eth0--->|breth0| -----> 172.18.0.3 OVN GR 100.64.0.4 --> join switch --> ovn_cluster_router --> 10.244.1.3 pod
+
+```
+
+### External Source -> Service -> Host Networked pod 
+
+This Scenario is a bit different, specifically traffic now needs to be directed from an external source to service and 
+then to the host itself(a host networked pod)
+
+In this flow, rather than going from breth0 into OVN we shortcircuit the path with physical flows on breth0 
+
+```text
+          host (ovn-worker, 172.18.0.3) 
+           ^
+           ^
+           |
+eth0--->|breth0| ---- 172.18.0.3 OVN GR 100.64.0.4 -- join switch -- ovn_cluster_router -- 10.244.1.3 pod
+
+```
+
+1. Match on the incoming traffic via it's nodePort, DNAT directly to the host networked endpoint, and send to `table=6` 
+
+```
+cookie=0x790ba3355d0c209b, duration=153.288s, table=0, n_packets=18, n_bytes=1468, idle_age=100, priority=100,tcp,in_port=1,tp_dst=<nodePort> actions=ct(commit,table=6,zone=64003,nat(dst=<nodeIP>:<targetIP>))
+```
+
+2. Send out the LOCAL ovs port on breth0, and traffic is delivered to host netwoked pod 
+
+```
+cookie=0x790ba3355d0c209b, duration=113.033s, table=6, n_packets=18, n_bytes=1468, priority=100 actions=LOCAL
+```
+
+3. Return traffic from the host networked pod to external source is matched in `table=7` based on the src_ip of the return 
+traffic being equal to `<targetIP>`, and un-Nat back to `<nodeIP>:<NodePort>`
+
+```
+ cookie=0x790ba3355d0c209b, duration=501.037s, table=0, n_packets=12, n_bytes=1259, idle_age=448, priority=100,tcp,in_port=LOCAL,tp_src=<targetIP> actions=ct(commit,table=7,zone=64003,nat)
+```
+
+4. Send the traffic back out breth0 back to the external source in `table=7`
+
+```
+cookie=0x790ba3355d0c209b, duration=501.037s, table=7, n_packets=12, n_bytes=1259, idle_age=448, priority=100 actions=output:1
+```
+
+## Host Traffic 
+
+This section will cover the networking entities hit when traffic travels from a cluster host via a service to either host
+networked pods or cluster networked pods
+
+### Host -> Service -> OVN Pod
+
+This case is similar to normal host -> Service -> OVN Pod except the SRC address of the Traffic will be that of the 
+management port `ovn-k8s-mp0` rather than the main address attatched to `eth0` on the host. 
```

Comment:
@astoycos I have few questions if you don't mind:

Is this change (use `ovn-k8s-mp0` IP instead of `breth0`'s) resulted from adding the externalTrafficPolicy feature?
Like normally the host to service traffic is routed through the `breth0` via the iptable rule:
```172.30.0.0/16 via 192.168.111.1 dev br-ex mtu 1400```